site/biome.jsonc

```diff
@@ -44,5 +47,6 @@
 				}
 			}
 		}
-	}
+	},
+	"$schema": "https://biomejs.dev/schemas/1.9.4/schema.json"
```

Comment:
Would this need to be manually bumped when @dependabot bumps biome version?

Reply:
I think so.

Reply:
unfortunately yeah, but it's worth having for the autocompletion help